### PR TITLE
Ownd 75 content changes to technical writing style guide page

### PR DESCRIPTION
**Pull request by cody-solomon**
#### Background information & Implementation Summary
We want to update the content on the _**[Writing Technical Content](https://styleguide.mailchimp.com/writing-technical-content/)**_ page of the Mailchimp Content Style Guide.

The new content is outlined in this [google doc](https://docs.google.com/document/d/1unkfKoEdiP8Axuy4CAh_w4kuh2QnSuKWSg74VwatnfA/edit#heading=h.3ln8tm114ci5).

This PR is made in junction with ([PR 9](https://github.com/freddieco/mc-style-guide-site/pull/9)). This PR deals with replacing the actual content of the page and also changed a link on another page to accurately reflect the new titling. See ([PR 9](https://github.com/freddieco/mc-style-guide-site/pull/9)) for other associated changes.

#### Contact information
Slack: @Cody Solomon or #attract-paid-media-squad
Email: cody.solomon@mailchimp.com

#### Ticket
https://jira.mailchimp.com/browse/OWND-75

### CODE REVIEW

**Comment by cody-solomon** [comment]
@stevenosloan So I can't add reviewers to this PR. I'm wondering if I may have set this up wrong. I also see that I don't have write access to merge, so is it possible I need some permissions?

EDIT: I'm also wondering if I pushed this to the wrong place lol. Investigating further.

**Comment by stevenosloan** [comment]
@cody-solomon looks like you got it spot on, this one is just super out of our current workflows now. This should probably warrant some discussion (again) since our content team no longer monitors this repo for outside feedback & updates if we should archive it as read only and vendor this content in the site build.

File: 08-writing-educational-content.html.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: How to Write Educational Content
+layout: article
+---
+
+At Mailchimp, educational content appears within our [Help Center](https://mailchimp.com/help/), as well as within our [Help Center](https://mailchimp.com/workshops/), certain marketing materials, and internal training documents. This section lays out the guiding principles for writing educational content, and discusses the main types of educational content that exist at Mailchimp.
```
Comment (stevenosloan) [inline 08-writing-educational-content.html.md]: second link here I think should be `[workshops](https://mailchimp.com/workshops/)`

Comment (cody-solomon) [reply 08-writing-educational-content.html.md]: Woops! Thanks for the catch! 🙏 
Fixed!

**Pull request merged**